test/functional/wallet_importmulti.py

```diff
@@ -63,6 +64,7 @@ def run_test(self):
         self.nodes[0].generate(1)
         self.nodes[1].generate(1)
         timestamp = self.nodes[1].getblock(self.nodes[1].getbestblockhash())['mediantime']
+        self.nodes[1].syncwithvalidationinterfacequeue()
```

Comment:
Just to make sure I understand correctly, what's the motivation for putting `syncwithvalidationinterfacequeue` after (instead of before) getting the `timestamp` value?

Reply:
getbestblockhash directly accesses the chainstate, so it is not needed there.

The wallet only gets the chainstate async over the validationinterface, so it is needed there.